lib/rom/sql/schema/column_inferrer.rb

```diff
@@ -0,0 +1,82 @@
+require 'rom/sql/types/pg'
```

Comment:
these pg extensions should be opt-in, inferrer class can be set in relation, so we can use different inferrers for different dbs by calling `schema_inferrer ROM::SQL::Schema::PGInferrer` in relation class